indico/migrations/versions/20220707_1007_b60f5c45acf7_add_option_to_enable_captcha.py

```diff
@@ -0,0 +1,25 @@
+"""Add option to registration forms to enable CAPTCHA for users without an account
+
+Revision ID: b60f5c45acf7
+Revises: 0c4bb2973536
+Create Date: 2022-07-07 10:07:49.004290
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'b60f5c45acf7'
+down_revision = '0c4bb2973536'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('forms', sa.Column('require_captcha', sa.Boolean(), nullable=False, server_default='true'), schema='event_registration')
```

Comment:
Default should be false, we don't want to enable this all of a sudden for existing regforms.